kvm/cpuid.go

```diff
@@ -1,3 +1,4 @@
+//nolint:dupl
```

Comment:
Didnt notice, we had an issue here. Maybe split this into a seperate commit?

Reply:
Thanks. It is indeed better to split the commit.